amplify/ts-code/src/db/Schemas.ts

```diff
@@ -1,98 +1,108 @@
 /**
- * @param id Name of item type. This needs to be unique.
- * @param description Optional description of item.
- * @param tags Tags to categorize item.
- * @param items List of IDs of all items of this item type.
- * @param id ID of item. User specified.
- */
+* @param id Name of item type. This needs to be unique.
+* @param description Optional description of item.
+* @param tags Tags to categorize item.
+* @param items List of IDs of all items of this item type.
+* @param id ID of item. User specified.
+*/
 export const MAIN_TABLE: string = process.env.STORAGE_MAIN_NAME
 export interface MainSchema {
-    id: string,
-    displayName: string,
-    description: string,
-    tags: string[],
-    items: string[]
+   id: string,
+   displayName: string,
+   description: string,
+   owner: string, // added owner field
+   location: string, // added loaction field
+   batch: string[], // added batch field
+   tags: string[],
+   items: string[]
 }
 
+
 /**
- * @param owner Name of the owner of the item or where the item is stored.
- * @param notes Notes specific to this item.
- * @param borrower Current borrower of item. Blank if available. Initialized as blank.
- * @param history List of entries in the history table
- * @param schedule List of entries in the schedule table [TODO: Implement Schedule]
- * @param batch List of batches this item is part of.
- */
+* @param owner Name of the owner of the item or where the item is stored.
+* @param notes Notes specific to this item.
+* @param borrower Current borrower of item. Blank if available. Initialized as blank.
+* @param history List of entries in the history table
+* @param schedule List of entries in the schedule table [TODO: Implement Schedule]
+* @param batch List of batches this item is part of.
+*/
 export const ITEMS_TABLE: string = process.env.STORAGE_ITEMS_NAME
-export interface ItemsSchema {
-    id: string,
-    name: string,
-    owner: string,
-    borrower: string,
-    batch: string[],
-    history: string[],
-    schedule: string[],
-    notes: string
+export interface ItemsSchema { // removed owner and batch fields
+   id: string,
+   name: string,
+   friendlyName: string, // added friendly name field
+   borrower: string,
+   borrowTime: string, // added borrow time field
+   returnTime: string, // added return time field
+   history: string[],
+   schedule: string[],
+   notes: string
 }
 
+
 export const TAGS_TABLE: string = process.env.STORAGE_TAGS_NAME
 export interface TagsSchema {
-    id: string,
```

Comment:
Can you fix all this white space changes? Seems unnecessary for the white space to change here and below.